### ISSUE TITLE
[UX] Add styling to configuration management diff

### ISSUE THREAD

**Issue by angelamnr** [backdrop/backdrop-issues#2042]
It would be nice if the configuration management diffs had styling similar to github. Right now, the config diffs are quite plain:
![Plain diff styling](http://imgur.com/8Vu840P.jpg)

I would like to add some CSS to the config module that makes it more apparent where your changes are, complete with fun colors:
![New styling changes](http://imgur.com/EPtKo3M.jpg)

I grabbed the red and green from error and confirmation message background colors elsewhere in Backdrop.

**Comment by quicksketch** [comment]
Hi @angelamnr! Welcome to the Backdrop issue queue!

This looks great!!

(I thought we already had an issue for this but I couldn't find it.)

I think the colors look great. They closely match the colors used in our alert messages (which are not theme-specific, they are general styling from `system.theme.css`). Thanks for the effort on this one. This has long been slightly embarrassing when showing others the diffs in Backdrop.

Diffs right now are only used in config.module, but could be used by contrib (e.g. diff.module). Maybe this should just be either A) part of `system.theme.css` (where message styling also is) or B) a whole new CSS file like `diff.theme.css` (and be in system.module's css directory). Maybe @wesruv or @jenlampton could weigh in here?

**Comment by quicksketch** [comment]
Other things that would be nice to address here also:

- The table vertical spacing is huge with Seven's default styling. Could we add a rule to Seven's `style.css` to reduce the table row padding?
- The `-` and `+` symbols are unnecessarily separated (left-to-right) from the line to which they are related. Could we reduce this padding to 0 so it's right next to the line? Again this might be Seven-specific.

**Comment by quicksketch** [comment]
I filed a follow-up PR at https://github.com/backdrop/backdrop/pull/1507 that includes your first one @angelamnr. The colors were really the hard part for me personally, but moving things around is easy. I implemented the above suggestions but all the colors are identical. Here's the dialog now in its full prettified glory:

![screenshot from 2016-08-31 21-26-52](https://cloud.githubusercontent.com/assets/100206/18155413/615fa480-6fc2-11e6-8119-e162be6013c4.png)
